x/src/cargo.rs

```diff
@@ -16,7 +17,17 @@ pub struct Cargo {
 
 impl Cargo {
     pub fn new<S: AsRef<OsStr>>(command: S) -> Self {
-        let mut inner = Command::new("cargo");
+        let mut inner = match env::var("CARGO") {
+            Ok(s) => Command::new(&s),
+            Err(_) => Command::new("cargo"),
+        };
+        if let Ok(s) = env::var("CARGOFLAGS") {
```

Comment:
clever again with cargo args - I'll prioritize getting a the logger pr in today since tracking what x will be doing will be even harder now.